db/models.py

```diff
@@ -50,3 +53,75 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
```

Comment:
i don't see user-related changes  in ` settings.py`
This is probably because you are on master branch. Shouldn't you work in develop branch instead?